stories/Components/TableStoriesDocs/TableDocs.mdx

```diff
@@ -1,42 +1,59 @@
 `import { Table } from "@bigbinary/neetoui";`
 
-We use Ant Design's `Table` component under the hood. For extra customization, refer [AntD Table](https://ant.design/components/table/#API).
```

Comment:
This line is actually needed. Otherwise we are not pointing out anywhere about this fact, but still will be linking to the AntD docs.